docs/how-to/add-model-and-api-endpoints.md

```diff
@@ -0,0 +1,151 @@
+# Add new model and API endpoints
```

Comment:
I think this file is already in docs/how-to/ . I don't know if you made updates to the text. Maybe just apply the updates to the other file?

The problem with having this file in this repo is that the code listings don't show up. This is because they're pointing to a separate repo which is the fang fork. It's better if the relevant lines could be extracted into markdown code blocks in this file so it'll look the same in any repo. This was something I started on but didn't finish doing.